src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -328,15 +330,16 @@ function MoneyRequestView({
     );
 
     const shouldShowMapOrReceipt = showMapAsImage || hasReceipt;
-    const shouldShowReceiptEmptyState = !hasReceipt && !isInvoice && (canEditReceipt || isAdmin || isApprover);
+    const isReceiptAllowed = !isPayReport && !isInvoice;
+    const shouldShowReceiptEmptyState = isReceiptAllowed && !hasReceipt && !isApproved && !isSettled && (canEditReceipt || isAdmin || isApprover);
     const noticeTypeViolations = transactionViolations?.filter((violation) => violation.type === 'notice').map((v) => ViolationsUtils.getViolationTranslation(v, translate)) ?? [];
     const shouldShowNotesViolations = !isReceiptBeingScanned && canUseViolations && ReportUtils.isPaidGroupPolicy(report);
 
     return (
         <View style={[StyleUtils.getReportWelcomeContainerStyle(isSmallScreenWidth, true, shouldShowAnimatedBackground)]}>
             {shouldShowAnimatedBackground && <AnimatedEmptyStateBackground />}
             <View style={shouldShowAnimatedBackground && [StyleUtils.getReportWelcomeTopMarginStyle(isSmallScreenWidth, true)]}>
-                {!isInvoice && (
+                {isReceiptAllowed && (
```

Comment:
We should show Receipt Audit Header only if the receipt is present of if we are showing receipt empty state. I reckon there will be the cases where `isReceiptAllowed` is true but there is no receipt or receipt empty state. Example case: Once the request without receipt is approved. What's your thought?

<img width="1440" alt="Screenshot 2024-05-13 at 14 05 22" src="https://github.com/Expensify/App/assets/25876548/c725ec3e-3c22-41fb-92cd-7c20ed4c9192">